src/briefcase/platforms/linux/appimage.py

```diff
@@ -175,13 +175,21 @@ class LinuxAppImageCreateCommand(
     def output_format_template_context(self, app: AppConfig):
         context = super().output_format_template_context(app)
 
-        # Add the manylinux tag to the template context.
         try:
-            tag = getattr(app, "manylinux_image_tag", "latest")
             manylinux_arch = {
                 "x86_64": "x86_64",
                 "i386": "i686",
-            }[LinuxDeploy.arch(self.tools.host_arch)]
+                "aarch64": "aarch64",
+            }[LinuxDeploy.arch(self.tools)]
+        except KeyError:
+            manylinux_arch = "unknown"
+            self.logger.warning(
+                f"There is not a manylinux base image for {LinuxDeploy.arch(self.tools)}"
```

Comment:
Slightly awkward wording; suggest:
```suggestion
                f"There is no manylinux base image for {LinuxDeploy.arch(self.tools)}"
```